packages/adapter/tsconfig.build.json

```diff
@@ -2,8 +2,10 @@
   "extends": "../../tsconfig.build.json",
   "compilerOptions": {
     "baseUrl": "./",
+    "outDir": "./dist",
     "rootDir": "src",
-    "outDir": "./dist"
+    // Enable at root tsconfig.json once all packages have been fixed
```

Comment:
Github should learn how to highlight this better :)

Edit: Actually, he understands tsconfig, but this is tsconfig.build so it's probably confused whether comments are allowed here.

Reply:
Yeah this is something I really don't like, but hoping to remove it soon